anndata/_io/h5ad.py

```diff
@@ -41,8 +51,6 @@ def write_h5ad(
     dataset_kwargs: Mapping = MappingProxyType({}),
     **kwargs,
 ) -> None:
-    from anndata.experimental import write_dispatched
```

Comment:
This is less likely to be a import with side effects, but it could be too: is it?

Reply:
also not necessary